src/pages/HelpPage.vue

```diff
@@ -17,22 +19,22 @@
         <card>
           <h2>Attacks Performed On Your Model</h2><hr>
```

Comment:
`<h2>` title here seems too large. Maybe `<h4>` is a better choice. Same for the first title above.